docs/pages/TextArea.md

```diff
@@ -0,0 +1,21 @@
+# TextArea
+
+The `TextArea` component is a styled `<textarea>` element.
+It accepts all native HTML attributes for the element.
```

Comment:
Nit: @unixchad, think this sentence could be removed that all React components are implied to accept HTML attributes, unless there's anything specific about `TextArea`, then this sentence could highlight it.

```suggestion
```

Reply:
done